NEWS.md

```diff
@@ -8,6 +8,8 @@
 
 * Setting `clean = TRUE` in `deploy_site_github()` removes old files from the deployed site before building a new one (#1297).
 
+* Markdown header attributes are now processed in all markdown files (@jonkeane)
```

Comment:
Can you please include the issue number in here? Sorry for not spotting earlier.

Reply:
There isn't a corresponding issue for this, but I can make one and reference it here if you would like.

Reply:
In that case, just use the PR number